src/gitops/components/application/ApplicationDetailsTitle.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+import DevPreviewBadge from '../../../components/import/badges/DevPreviewBadge';
+import { DEFAULT_NAMESPACE } from '../../utils/constants';
+import { isApplicationRefreshing } from '../../utils/gitops';
+import { useGitOpsTranslation } from '../../utils/hooks/useGitOpsTranslation';
+import { Action, K8sModel, K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { Breadcrumb, BreadcrumbItem, Spinner, Title } from '@patternfly/react-core';
+import ActionsDropdown from '../../utils/components/ActionDropDown/ActionDropDown';
+import DetailsPageTitle, { PaneHeading } from '../../utils/components/DetailsPageTitle/DetailsPageTitle';
+import './application-details-title.scss';
+
+type ApplicationPageTitleProps = {
+  obj: K8sResourceCommon;
+  model: K8sModel;
+  name: string;
+  namespace: string;
+  actions: Action[];
+};
+
+const ApplicationDetailsTitle: React.FC<ApplicationPageTitleProps> = ({
+  obj,
+  model,
+  name,
+  namespace,
+  actions,
+}) => {
+  const { t } = useGitOpsTranslation();
+  
+  // Determine the correct icon text and styling based on the model
+  const isApplicationSet = model.kind === 'ApplicationSet';
+  const iconText = isApplicationSet ? 'AS' : 'A';
```

Comment:
Maybe provide this as properties?  We will have another details page for app projects.